README.md

```diff
@@ -1 +1,155 @@
+- **Understand the use of the .gitignore folder**
+- **How to write a descriptive and easy to understand README in business English**
+- **Apply the use of HTML with a simple header**
+- **Apply the use of CSS with a class applied to the header created with HTML**
+- **How to open pull requests in GitHub**
+- **How to write short, detailed, easy to understand descriptions of pull requests**
+- **Demonstrate an ability to submit a project for code review**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Did you know...? <a name="did-you-know"></a>
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need:
+
+- A code editor
+- A web browser
+- Basic management of the console
+- Basic knowledge of HTML and CSS
+- Basic knowledge of Git and GitHub
+
+### Clone <a name="clone"></a>
+
+Clone this repository to your desired folder:
+
+cd "your-folder"
+
+git clone https://github.com/Otromascomotu/Hello-Microverse/tree/implementing-requirements-of-hello-microverse-project
+
+### Linters <a name="linters"></a>
+
+To fix linters errors, run the following commands, accordingly:
+
+<details>
+  <summary>Webhint</summary>
+  <ul>
+    <li>Run npx hint . to fix validation errors</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Stylelint</summary>
+  <ul>
+    <details>
+     <summary>No saas</summary>
+     <ul>
+       <li>Run npx stylelint "**/*.{css,scss}" on the root of your directory of your project to fix linter errors</li>
+     </ul>
+    </details>
+    <details>
+      <summary>With saas</summary>
+      <ul>
+        <li>In the file .github/workflows/linters.yml line 48 replace "**/*.{css,scss}" with "**/*.scss"</li>
+        <li>Run npx stylelint "**/*.scss" to lint you SCSS files (and not the generated CSS) to fix linter errors</li>
+      </ul>
+    </details>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="author"></a>
+
+👤 **Luis Laverde**
+
+- GitHub: [@otromascomotu](https://github.com/otromascomotu)
+- Twitter: [@otromascomotu](https://twitter.com/otromascomotu)
+- LinkedIn: [/luis-laverde-developer](https://linkedin.com/in/luis-laverde-developer/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
+
+-To Microverse for giving me the opportunity to apply what I've learned and showing me the way to become a good programmer.
+
+-To my teacher and friend: Adrián Gonzalez, who taught me the basics of code, good practices and my love for code.
+
+-To my wife for encouraging me so much!
+
+-To myself, by my will, desire and effort of innumerable days, nights and early mornings.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly setup your linters correctly its mandatory for better Microverse guildline how to setup linters you can follow [link](https://github.com/microverseinc/linters-config/tree/master/html-css)

Reply:
I ran the commands of Webhint and Stylelint linters and was able to resolve 3 errors from the .css file:

- missing indent to only 2 spaces instead of 4 spaces,
- missing semicolon at the end of line 2, and 
- missing an extra blank line at the end of all css code 

Then, all was good but I forgot to push these changes before sending you the second pull request again... how should I do? push the changes immediately to join the branch or wait for a second revision and then push them to open a third revision?

Reply:
Finally push the changes in the css so that the project is complete, without errors!